doc/source/release-notes/Percona-Server-5.7.19-17.rst

```diff
@@ -0,0 +1,73 @@
+.. rn:: 5.7.19-17
+
+========================
+Percona Server 5.7.19-17
+========================
+
+Percona is glad to announce the release of Percona Server 5.7.19-17
+on August 30, 2017.
+Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.7/Percona-Server-5.7.19-17/>`_
+and from the :doc:`Percona Software Repositories </installation>`.
+
+This release is based on `MySQL 5.7.19
+<http://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-19.html>`_
+and includes all the bug fixes in it.
+Percona Server 5.7.19-17 is now the current GA release in the 5.7 series.
+All software developed by Percona is open-source and free.
+Details of this release can be found in the `5.7.19-17 milestone on Launchpad
+<https://launchpad.net/percona-server/+milestone/5.7.19-17>`_.
+
+.. note:: Red Hat Enterprise Linux 5 (including CentOS 5 and other derivatives),
+   Ubuntu 12.04 and older versions are no longer supported by Percona software.
+   The reason for this is that these platforms reached end of life,
+   will not receive updates and are not recommended for use in production.
+
+New Features
+============
+
+* Included the :ref:`Percona MyRocks storage engine <myrocks_intro>`
+
+  .. note:: MyRocks for Percona Server is currently experimental
+     and not recommended for production deployments until further notice.
+     You are encouraged to try it in a testing environment
+     and provide feedback or report bugs.
+
+* :bug:`1708087`: Added the ``mysql-helpers`` script
+  to handle checking for missing ``datadir`` during startup.
+  Also fixes :bug:`1635364`.
+
+Platform Support
+================
+
+* Stopped providing packages for Ubuntu 12.04 due to its end of life.
```

Comment:
"Starting with this release ... " ?

Reply:
Yes, these are release notes for this release, so anything in it applies to this release.